plumbing/object/commit.go

```diff
@@ -233,6 +233,16 @@ func (b *Commit) Encode(o plumbing.EncodedObject) error {
 	return b.encode(o, true)
 }
 
+// PGPPayload export to a Reader the commit payload that is signed
+func (b *Commit) PGPPayload() (io.Reader, error) {
```

Comment:
I rather keep the pattern of `Encode` and just expose `EncodeWithoutSignature` for example.

Reply:
done